Shuffle.cs

```diff
@@ -660,7 +660,10 @@ private void PlaceItem(int CurrentItem, List<int> Targets)
             if (ItemList[CurrentItem].Replaces != -1)
             {
                 return;
-            };
+            }
+
+			int attempts = 0;
```

Comment:
Would be better not to mix tabs and spaces for indentation. Issue is present throughout this pull request.